docs/runbook/maintain-cardano-min-versions-file/README.md

```diff
@@ -0,0 +1,14 @@
+#  Cardano minimum versions for Signers
+
+## Introduction
+
+The [cardano-min-versions.json](../../../cardano-min-versions.json) file, which specifies the minimum supported versions of the Cardano node for Mithril signers
+This ensures compatibility and functionality across different networks.
+
+## Maintaining the file
+
+When a new a new Cardano node version is released:
+- Verify the Cardano node compatibility with Mithril nodes, especially through the CI process.
```

Comment:
This is not very clear:
- the process to check compatibility is to create a PR with the new version of the node tested in the `devnet` with the CI e2e test
- then 2 cases:
  - compatible without modification on Mithril nodes (in that case bump versions right away)
  - or modifications required by the Mithril team (in that case bump version after new distribution is released)

Reply:
Done but I'd like your opinion again please 🙏.